src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -412,6 +533,47 @@ public void setValues(JSONObject formData) {
         replicationConfig = ReplicationConfig.createReplicationConfigFromJSON(formData);
     }
 
+    /**
+     * Enrich the gerrit commands with the additional labels, if present.
+     */
+    private void initializeGerritCommands() {
+        List<String> stringLabels = new ArrayList<String>(Arrays.asList("verified", "code-review"));
+
+        for (VerdictCategory label : categories) {
+            stringLabels.add(label.getVerdictValue().toLowerCase());
+
+            if (!label.getVerdictValue().equalsIgnoreCase(CODE_REVIEW) && !label.getVerdictValue().equalsIgnoreCase(VERIFIED)) {
+                String commandAddition = " --" + label.getVerdictValue().toLowerCase() + " <" + label.getVerdictValue().toUpperCase() + "> ";
+                gerritVerifiedCmdBuildStarted = gerritVerifiedCmdBuildStarted.contains(commandAddition)
+                    ? gerritVerifiedCmdBuildStarted : (gerritVerifiedCmdBuildStarted += commandAddition);
+
+                gerritVerifiedCmdBuildSuccessful = gerritVerifiedCmdBuildSuccessful.contains(commandAddition)
+                    ? gerritVerifiedCmdBuildSuccessful : (gerritVerifiedCmdBuildSuccessful += commandAddition);
+
+                gerritVerifiedCmdBuildFailed = gerritVerifiedCmdBuildFailed.contains(commandAddition)
+                    ? gerritVerifiedCmdBuildFailed : (gerritVerifiedCmdBuildFailed += commandAddition);
+
+                gerritVerifiedCmdBuildUnstable = gerritVerifiedCmdBuildUnstable.contains(commandAddition)
+                    ? gerritVerifiedCmdBuildUnstable : (gerritVerifiedCmdBuildUnstable += commandAddition);
+
+                gerritVerifiedCmdBuildNotBuilt = gerritVerifiedCmdBuildNotBuilt.contains(commandAddition)
+                    ? gerritVerifiedCmdBuildNotBuilt : (gerritVerifiedCmdBuildNotBuilt += commandAddition);
+            }
+        }
+        String gerritCmdRegex = "(--([\\S]+) (<[\\S]+>)){1}";
+        Pattern pattern = Pattern.compile(gerritCmdRegex);
+        Matcher matcher = pattern.matcher(gerritVerifiedCmdBuildStarted);
+        while (matcher.find()) {
+            if (!stringLabels.contains(matcher.group(2))) {
```

Comment:
Perhaps I'm misreading  the code, but group 1 contains the verdictValue.lowercase and group 2 the uppercase version. Will this ever match?